tests/invalid.rs

```diff
@@ -0,0 +1,108 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use steward::{app, init_tracing, BUNDLE, PKCS10};
+
+mod common {
+    pub mod validation;
+}
+use common::validation::{certificates_state, cr, hostname_state};
```

Comment:
What is this trying to achieve?

Reply:
This makes common functions available to the separate unit tests, based on how Tokio's [rt_panic.rs](https://github.com/tokio-rs/tokio/blob/master/tokio/tests/rt_panic.rs) uses [support/panic.rs](https://github.com/tokio-rs/tokio/blob/master/tokio/tests/support/panic.rs).

Reply:
File deleted